### DISCUSSION TITLE
Dynamic stats bug

### DISCUSSION THREAD

**Discussion by Fillox112** [betfinio/public#97] [Bugs and Reports]
<img width="542" alt="Snímek obrazovky 2024-11-24 v 18 53 15" src="https://github.com/user-attachments/assets/1cf9bc26-0fa2-477d-a49d-54c7c7af2c5a">

There is icon for the conservative (vault) in the dynamic stats window

also its showing that the roulette did 0

**Comment by bf-monster** [comment]
Hello, missing data, thank you, we will fix as soon as possible
